example/src/main/java/org/xmtp/android/example/pushnotifications/PushNotificationTokenManager.kt

```diff
@@ -37,18 +37,22 @@ object PushNotificationTokenManager {
     }
 
     fun ensurePushTokenIsConfigured() {
-        FirebaseMessaging.getInstance().token.addOnCompleteListener(OnCompleteListener { request ->
-            if (!request.isSuccessful) {
-                Log.e(TAG, "Firebase getInstanceId() failed", request.exception)
-                return@OnCompleteListener
-            }
-            request.result?.let {
-                if (xmtpPushState.value is XMTPPushState.Ready) {
-                    xmtpPush.register(it)
-                    configureNotificationChannels()
+        try {
+            FirebaseMessaging.getInstance().token.addOnCompleteListener(OnCompleteListener { request ->
+                if (!request.isSuccessful) {
+                    Log.e(TAG, "Firebase getInstanceId() failed", request.exception)
+                    return@OnCompleteListener
+                }
+                request.result?.let {
+                    if (xmtpPushState.value is XMTPPushState.Ready) {
+                        xmtpPush.register(it)
+                        configureNotificationChannels()
+                    }
                 }
-            }
-        })
+            })
+        } catch (e: Exception) {
+            Log.e(TAG, "Firebase not setup", e)
+        }
```

Comment:
The example app was broken so I pushed up a small fix that will allow people to more easily build the example app without notifications.